pkg/proxy/iptables/proxier_test.go

```diff
@@ -1017,15 +863,31 @@ func TestClusterIPReject(t *testing.T) {
 	)
 	fp.syncProxyRules()
 
-	svcChain := string(servicePortChainName(svcPortName.String(), strings.ToLower(string(v1.ProtocolTCP))))
-	svcRules := ipt.GetRules(svcChain)
-	if len(svcRules) != 0 {
-		errorf(fmt.Sprintf("Unexpected rule for chain %v service %v without endpoints", svcChain, svcPortName), svcRules, t)
-	}
-	kubeSvcRules := ipt.GetRules(string(kubeServicesChain))
-	if !hasJump(kubeSvcRules, iptablestest.Reject, svcIP, svcPort) {
-		errorf(fmt.Sprintf("Failed to find a %v rule for service %v with no endpoints", iptablestest.Reject, svcPortName), kubeSvcRules, t)
-	}
+	expected := `*filter
```

Comment:
`dedent.Dedent()` might make these more readable?

Reply:
I tried it; it's not _much_ more readable, and it makes it harder to update the expected results when they change... so I think no?

I did at least fix it so you can do 

```
        expected := `
*filter
blah blah blah
```

instead of

```
        expected := `*filter
blah blah blah
```